recipes/rift/meta.yaml

```diff
@@ -0,0 +1,123 @@
+{% set name = "RIFT" %}
+{% set version = "0.0.14.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 37edc652de6787be884b1b638e8f41315b0c9ac4fc11a92f9d61d6530991883d
+
+build:
+  number: 0
+  skip: true  # [win or py>30]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - corner
+    - gwdatafind
+    - h5py
+    - lalsuite
+    - ligo-segments
+    - matplotlib
+    - numba
+    - numpy
+    - pip
+    - python
+    - python-ligo-lw
+    - scikit-learn
+    - setuptools
+
+test:
+  imports:
+    - RIFT
+    - RIFT.integrators
+    - RIFT.interpolators
+    - RIFT.likelihood
+    - RIFT.physics
+  commands:
+    # set environment
+    - export EOB_ARCHIVE="${HOME}"
+    - export EOB_BASE=""
+    - export EOB_C_BASE=""
+    - export EOB_C_ARCHIVE="${HOME}"
+    - export EOB_C_ARCHIVE_NMAX="1"
+    - export EOS_TABLES="${HOME}"
+    - export GW_SURROGATE=""
+    - export LALSIMULATION_DATADIR="${PREFIX}/share/lalsimulation"
+    - export MATLAB_BASE=""
+    - export MPLBACKEND="agg"
```

Comment:
Should you add an activation script that will set does env variables for you when someone installs this package?
Or is it just for testing?